### PR TITLE
Updated intro content

### PR DESCRIPTION
**Pull request by mishaone**
RHDEVDOCS-438: added to original draft of this content with additional information from various sources in JIRA

Robert, Preeti: could either (or both, if you have time) of you review this please?

### CODE REVIEW

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc [hunk 1]
```diff
@@ -5,15 +5,26 @@
...
 * Project planning and issue tracking using your choice of Agile methodology.
 * Creating development environments and testing requirements.
-* Coding using an integrated development environment based on Eclipse Che.
-* Building, testing, deploying, and monitoring codebases using pipelines.
+* Coding using an integrated development environment based on Eclipse Che, which provides a workspace based on a containerized version of your application.
+* Development in this context-aware workspace provides the appropriate context for the developer and project, such as which branch, repository, language, and files are referenced.
+* Building, testing, deploying, and monitoring codebases using Continuous Integration pipelines that can be tailored to the requirements of a team or an individual.
+* Using integrated but customizable Jenkins build pipelines to facilitate rapid testing and environment promotion.
 * Interacting with integrated GitHub repositories.
+* Deploying applications packaged as a set of containers to OpenShift, which powers the underlying services, such as the Eclipse Che workspaces, Work Items, and Pipelines.
```
Comment (rkratky) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: WIs are powered by OpenShift?

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc [hunk 2]
```diff
@@ -5,15 +5,26 @@
...
 * Compiling and reviewing code in Pull Requests as required.
 * Automating releases of development projects.
+* Use the full-stack analysis feature on your project's included open source libraries to identify potential security issues and recommend safer alternative libraries or library versions to use instead.
```
Comment (rkratky) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: `s/open source/open-source/`

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc [hunk 3]
```diff
@@ -5,15 +5,26 @@
...
 
-{osio} offers features for seamless management of end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
+{osio} offers features to seamlessly manage end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
 
 * Project planning and issue tracking using your choice of Agile methodology.
 * Creating development environments and testing requirements.
-* Coding using an integrated development environment based on Eclipse Che.
-* Building, testing, deploying, and monitoring codebases using pipelines.
+* Coding using an integrated development environment based on Eclipse Che, which provides a workspace based on a containerized version of your application.
+* Development in this context-aware workspace provides the appropriate context for the developer and project, such as which branch, repository, language, and files are referenced.
```
Comment (ldimaggi) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: Awkward wording - can we replace the 2nd instance of context with a different word, such as "environment?"

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 1 above)
Comment (ldimaggi) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: Robert is correct in that the workitems are not powered by OpenShift. Suggested wording:

Deploying applications packaged as a set of containers to OpenShift, which powers the underlying services, such as the Eclipse Che workspaces and Pipelines. Development and test tasks are defined by workitems in the integrated Planner.

Comment (Preeticp) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: which powers the underlying services>which is powered by the underlying services

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc [hunk 4]
```diff
@@ -5,15 +5,26 @@
...
+* Using integrated but customizable Jenkins build pipelines to facilitate rapid testing and environment promotion.
 * Interacting with integrated GitHub repositories.
+* Deploying applications packaged as a set of containers to OpenShift, which powers the underlying services, such as the Eclipse Che workspaces, Work Items, and Pipelines.
 * Compiling and reviewing code in Pull Requests as required.
 * Automating releases of development projects.
+* Use the full-stack analysis feature on your project's included open source libraries to identify potential security issues and recommend safer alternative libraries or library versions to use instead.
+* Use the self-learning analytics engine to improve the code quality and make data-based choices. The engine scans community libraries, CVEs, commit authors, community size, and other metrics to provide useful real-time assistance for developers that learns and improves over time.
+* Use the QuickStart wizards to create basic applications with their own new repo on GitHub, a build pipeline integrated with Jenkins, and a related new OpenShift project.
+* Using migration wizards to migrate legacy J2EE applications to the cloud.
```
Comment (Preeticp) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: We have 14 bullet points in this list and quite wordy ones. Can we think of grouping these into 4/5 sections and have bullets within them to retain user attention. Having sections with titles would help an user to skim over the information if needed and read details where required.

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 1 above)
Comment (Preeticp) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: which powers the underlying services>which is powered by the underlying services?

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 2 above)
Comment (Preeticp) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: Suggest replacing Libraries with Components as that is the word going to be used on the UI

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc [hunk 5]
```diff
@@ -5,15 +5,26 @@
...
+* Building, testing, deploying, and monitoring codebases using Continuous Integration pipelines that can be tailored to the requirements of a team or an individual.
+* Using integrated but customizable Jenkins build pipelines to facilitate rapid testing and environment promotion.
 * Interacting with integrated GitHub repositories.
+* Deploying applications packaged as a set of containers to OpenShift, which powers the underlying services, such as the Eclipse Che workspaces, Work Items, and Pipelines.
 * Compiling and reviewing code in Pull Requests as required.
 * Automating releases of development projects.
+* Use the full-stack analysis feature on your project's included open source libraries to identify potential security issues and recommend safer alternative libraries or library versions to use instead.
+* Use the self-learning analytics engine to improve the code quality and make data-based choices. The engine scans community libraries, CVEs, commit authors, community size, and other metrics to provide useful real-time assistance for developers that learns and improves over time.
```
Comment (Preeticp) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: suggest replacing CVEs with Security vulnerabilities.
It does not check for community authors. Also Community size can not be determined by number of Watchers on GH/stars etc.
Suggest making it generic-The engine scans component level security vulnerabilities and other git hub project metrics to provide...

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc [hunk 6]
```diff
@@ -5,15 +5,26 @@
...
 == Features
 
-{osio} offers features for seamless management of end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
+{osio} offers features to seamlessly manage end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
 
 * Project planning and issue tracking using your choice of Agile methodology.
 * Creating development environments and testing requirements.
-* Coding using an integrated development environment based on Eclipse Che.
-* Building, testing, deploying, and monitoring codebases using pipelines.
+* Coding using an integrated development environment based on Eclipse Che, which provides a workspace based on a containerized version of your application.
```
Comment (Preeticp) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: a workspace based on a containerized version of your application.> Afaiu, the workspace sits in a container, using which we create a containerized application.
On the other hand by 'containerized version of your application' are you suggesting 'containerized template for your application'?

**Comment by fabric8cd** [comment]
@mishaone snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-14-1 is available for testing at http://fabric8-online-docs-app-pr-14-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@mishaone snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-14-2 is available for testing at http://fabric8-online-docs-app-pr-14-fabric8-online-docs.badger.fabric8.io

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 2 above)
Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: I was sure I'd seen the non-hyphenated version being used so I consulted the docs conventions and it looks like the hyphenated version is discouraged: https://doc-stage.usersys.redhat.com/documentation/en-us/ccs_internal_documentation/1.0/html-single/glossary_of_terms_and_conventions_for_product_documentation/#o

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 1 above)
Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: changed to exclude WIs

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 3 above)
Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: s/workspace/environment

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 1 above)
Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: Removed work items, as above. The second suggestion goes to passive tense and also changes the meaning of the sentence here, which was intended to be that OS powers the underlying services provided by OSIO (Che workspace, etc.)

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 2 above)
Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: Changed to include the recommendations.

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 5 above)
Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: Updated as recommended.

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 6 above)
Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: Updated to this: "Coding using an integrated development environment based on Eclipse Che, which provides a workspace for developing your application."

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc (hunk 4 above)
Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: Regrouped, looking for feedback on whether the grouping makes sense.

**Comment by mishaone** [comment]
Moved to https://github.com/fabric8io/fabric8-online-docs/pull/26

**Comment by rkratky** [comment]
@mishaone, what's the  status of this PR?

**Comment by mishaone** [comment]
@rkratky okay to merge this, but I lost the local copy of this and copied over the changes in a new PR (https://github.com/fabric8io/fabric8-online-docs/pull/26) which is now closed. It should be okay to just close this PR without merging it since the changes are merged already in 26.

**Pull request closed without merging**